Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -787,6 +794,33 @@ extension ProjectDescription.TargetDependency {
 
         return targetDependencies + linkerDependencies
     }
+    
+    private static func frameworkStatus(
+        name: String,
+        platform: ProjectDescription.Platform,
+        deploymentTarget: ProjectDescription.DeploymentTarget
+    ) -> ProjectDescription.SDKStatus {
+        guard
+            let data = try? Data(contentsOf: URL(
+                string: "https://developer.apple.com/tutorials/data/documentation/\(name.camelCaseToSnakeCase()).json")!
+            ),
```

Comment:
I wouldn't do a network call here, I would rather:
1. always map to optional if it's not a problem
2. have a static map of apple framework to their min iOS version, there could be a bit of effort to keep it updated but there aren't many of them

@kwridan @fortmarek do you think `1` could be a problem in some case?

Reply:
I don't completely understand the functionality – is this something SPM does? Or are we adding something extra here? Seems weird we would allow a framework that has a higher target than the one we're integrating it in.

Reply:
Yes, SPM allows that.
An example is Adjust that uses some mattering related frameworks if available, or does what it can without them 😁
Unfortunately in the Package.swift there is no mention of required or not required, it's just some SPM magic

Reply:
Oof, then I'd be benevolent and use `.optional`. This might crash in runtime if the framework is actually needed but if we don't know what logic SPM uses, I feel we should be less strict.